pkg/odo/cli/component/common_push.go

```diff
@@ -53,14 +67,33 @@ func (cpo *CommonPushOptions) ResolveSrcAndConfigFlags() {
 	}
 }
 
+func (cpo *CommonPushOptions) ValidateComponentCreate() error {
```

Comment:
please add some comments to the function

Reply:
Added